README.md

```diff
@@ -77,10 +77,12 @@ $ aegir-test browser
 $ gulp test:browser
 ```
 
-If you want to run tests in a webworker you can also pass `--webworker` as a flag to enable that.
+Browser tests by default run inside the main thread and a webworker thread. If you want to disable or only run your tests in a webworker, you can do it by passing the flag:
 
-If the needed environment variables are set, tests are also run on [Sauce Labs].
-You will need
+- `--webworker=false`
+- `--webworker=only`
```

Comment:
I am not saying the current version is particularly good, in reality the whole thing just needs refactoring and clean up, this was never intended to support many options. But this PR for me doesn't actually fix the issues which is why I am hesitant to merge it.